schematic/visualization/tangled_tree.py

```diff
@@ -784,35 +864,37 @@ def get_ancestors_nodes(self, subgraph, components):
 
         return all_parent_children
 
-    def get_tangled_tree_layers(self, save_file=True):
+    def get_tangled_tree_layers(self, save_file: bool = True):
         """Based on user indicated figure type, construct the layers of nodes of a tangled tree.
         Inputs:
             save_file (bool): Indicates whether to save a file locally or not.
         Outputs:
             all_layers (list of json strings):
-                If save_file == False: Each string represents contains the layers for a single tangled tree.
+                If save_file == False: Each string represents contains the layers
+                  for a single tangled tree.
                 If save_file ==True: is an empty list.
 
         Note on Dependency Tangled Tree:
-            If there are many conditional requirements associated with a depependency, and those
+            If there are many conditional requirements associated with a dependency, and those
             conditional requirements have overlapping attributes associated with them
             the tangled tree will only report one
 
         """
+        # pylint: disable=too-many-locals
         # Gather the data model's, topological generations, nodes and edges
-        topological_gen, nodes, edges, subg = self.get_topological_generations()
+        topological_gen, nodes, edges, subgraph = self.get_topological_generations()
 
         if self.figure_type == "component":
             # Gather all source nodes
             source_nodes = self.find_source_nodes(nodes, edges)
 
             # Map all children to their parents and vice versa
-            child_parents, parent_children = self.get_parent_child_dictionary(
-                nodes, edges
-            )
+            child_parents, parent_children = self.get_parent_child_dictionary(edges)
```

Comment:
curious about how these args were passed before then now. To be safe lets set this as `self.get_parent_child_dictionary(edges=edges)`